### PR TITLE
bootnodes as empty string does not work, needs to be a list

### PR DESCRIPTION
**Pull request by macfarla**
comment out node.bootnodes="" since it needs to be a list

### CODE REVIEW

File: config/config.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ port = 9000
 # "static" - no discovery, only connect to static peers
 # "discv5" - Enable discovery v5
 discovery = "static"
-bootnodes = ""
+#bootnodes = ""
```
Comment (ajsutton) [inline config/config.toml:12]: Probably should just be 
```suggestion
bootnodes = []
```

**Review by ajsutton — APPROVED** [review]
Yeah sorry about that.  Thought I'd updated it.

File: config/config.toml (hunk 1 above)
Comment (macfarla) [reply config/config.toml:12]: I tried that, it didn't work :/

Comment (ajsutton) [reply config/config.toml:12]: hmm, it did for Cem and I.  I just use a custom config file so forgot to update the checked in version.

Comment (rolfyone) [reply config/config.toml:12]: it really should be defaulted in ArtemisConfiguration so that removal doesn't tank everyones instances...

**Pull request merged**